### ISSUE TITLE
Edit button should not be visible in embedded player below certain screen size

### ISSUE THREAD

**Issue by kschzt** [vizorvr/vizor#1561]

**Comment by gmarinov** [comment]
what is the certain screen size?

**Comment by kschzt** [comment]
@gmarinov maybe this should be "don't open the editor inside an embed"

**Comment by esaruoho** [comment]
yeah, "disable edit-button showing inside embed" would be very good. I've seen this asked for:
https://vizorvr.slack.com/archives/C0D7S35JQ/p1499074197036923

any thoughts on how difficult this would be to do, @gmarinov ?
